spring-boot-project/spring-boot-docs/src/docs/antora/modules/reference/pages/actuator/loggers.adoc

```diff
@@ -31,3 +31,16 @@ To configure a given logger, `POST` a partial entity to the resource's URI, as t
 ----
 
 TIP: To "`reset`" the specific level of the logger (and use the default configuration instead), you can pass a value of `null` as the `configuredLevel`.
+
+== OpenTelemetry
+By default, the OpenTelemetry SDK logs are not configured. You can provide the location of the OpenTelemetry logs endpoint to configure it:
+
+[source,yaml]
+----
+management:
+  otlp:
+    logging:
+      endpoint: "https://otlp.example.com:4318/v1/logs"
+----
+
+Note: The OpenTelemetry Logback appender and Log4j appender are not part of Spring Boot. You have to provide and configure them yourself. For more details, https://github.com/spring-projects/spring-boot/pull/40961[see this pull request].
```

Comment:
`Note` should be `NOTE` and we prefer one sentence per line.

I'm not sure that we should link to the pull request. Perhaps we can say "For more details, see the Logback and Log4j2 appenders in the https://github.com/open-telemetry/opentelemetry-java-instrumentation[OpenTelemetry GitHub repository]

Reply:
the pull request link add informtion on how to link the appender to the OpenTelemetry sdk.

Can I add this sample to the documentation?
```java
@Bean
public ApplicationListener<ApplicationReadyEvent> logbackOtelAppenderInitializer(OpenTelemetry openTelemetry) {
    return event -> OpenTelemetryAppender.install(openTelemetry);
}
```

Reply:
If you want to check the docs, you should be able to run `./gradlew antora` then `open spring-boot-project/spring-boot-docs/build/site/index.html`.

Reply:
It's not a good idea. The `OpenTelemetryAppende`r is not in the classpath, and it broke the build. I changed the code sample into two tips, and they do the job.